test/test_syncer/test_project_syncer.py

```diff
@@ -195,3 +195,27 @@ def test_sync_api_integration(self, mock_cat):
         res = self.app.put("/api/project/{}".format(project.id), headers=headers, data=json.dumps(payload))
         assert res.status_code == 200, 'build_payload output should result in valid api query'
         assert res.json["info"]["sync"] == payload["info"]["sync"]
+
+    @with_context
+    @patch('pybossa.syncer.project_syncer.ProjectSyncer.get_target')
+    @patch('pybossa.syncer.category_syncer.CategorySyncer.get_target', return_value={'id': 1})
+    @patch('pybossa.syncer.project_syncer.ProjectSyncer.cache_target')
+    @patch('pybossa.syncer.project_syncer.ProjectSyncer._sync')
+    @patch('pybossa.syncer.requests')
+    def test_sync_update_existing(self, fake_requests, mock_update, mock_cache, mock_cat, mock_get):
+        project_syncer = ProjectSyncer(self.target_url, self.target_key)
+        user = UserFactory.create(admin=True, email_addr=u'user@test.com')
+        project_syncer.syncer = user
+
+        res = Response()
+        res.ok=False
```

Comment:
Should be `res._ok` and `res._content`, see [here](https://github.com/bloomberg/pybossa/blob/07aae932e7fa9e2bdadda0d9f2613022906d93bd/test/test_syncer/test_project_syncer.py#L33)